code/edge/src/test/java/com/adobe/marketing/mobile/RequestBuilderTest.java

```diff
@@ -315,6 +317,74 @@ public void getPayloadWithExperienceEvents_addsKonductorConfigWithStreaming_when
 		assertEquals("\n", payloadMap.get("meta.konductorConfig.streaming.lineFeed"));
 	}
 
+	@Test
+	public void getPayloadWithExperienceEvents_setsSdkConfigMeta_whenSdkConfigPresent() throws Exception {
+		List<Event> events = getSingleEvent(getExperienceEventData("value", "5dd603781b95cc18a83d42ce"));
```

Comment:
nit: can we skip the datasetId setting for these tests unless the intention is to test both config overrides + dataset id